app/src/main/res/layout-sw600dp/help_activity.xml

```diff
@@ -0,0 +1,105 @@
+<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/help_activity_drawer_layout"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <include
+      android:id="@+id/help_activity_toolbar"
+      layout="@layout/toolbar" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/oppiaGreyBackground">
+
+      <FrameLayout
+        android:id="@+id/help_fragment_placeholder"
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/options_separator"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+      <ImageButton
+        android:id="@+id/help_multipane_options_back_button"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:layout_marginStart="12dp"
+        android:layout_marginTop="12dp"
+        android:background="@color/oppiaGreyBackground"
+        android:contentDescription="@string/back_button"
```

Comment:
use `navigate_up`

Reply:
Done.

Reply:
Also please note this https://github.com/oppia/oppia-android/pull/3671#pullrequestreview-729628815